drone-hugo.sh

```diff
@@ -15,18 +16,28 @@ PLUGIN_THEME=${PLUGIN_THEME:-"false"}
 PLUGIN_URL=${PLUGIN_URL:-"false"}
 PLUGIN_VALIDATE=${PLUGIN_VALIDATE:-"false"}
 
+# The hugo command
+HUGO_COMMAND="hugo"
+
 function addArgument {
   echo $1
   HUGO_COMMAND="${HUGO_COMMAND} $1"
 }
 
-# Hugo Command
-HUGO_COMMAND="hugo"
+# Download hugo binary if version attribute is set. When not set, use the one shipped binary inside container
+if [[ $PLUGIN_VERSION != "false" ]] ; then
+  echo "Downloading hugo version v${PLUGIN_VERSION}..."
```

Comment:
You could also output the latest version and tell the user that a newer one is available compared to the one they have in their YAML.

Reply:
Good idea, than everybody reading the output can directly see that he's using an outdated version.

Reply:
Do you think a simple hint of the version used in container like 

```
$ hugo version
Hugo Static Site Generator v0.25.1 linux/amd64 BuildDate: 2017-08-03T10:06:42+02:00
```

in the pipeline output would be enough? Or do you suggest some string formatting and comparing the actual versions?

Reply:
If the formatting is easy just do it, otherwise we can delay it until the go-based plugin version.

Reply:
I would say let's delay it then. I think it's more handy to do this in a go function instead of implementing fancy formatting logic in the bash script. I can create an issuse for that as a reminder.